SPECS/xs-openssl.spec

```diff
@@ -539,6 +552,25 @@ export LD_LIBRARY_PATH
 %postun libs -p /sbin/ldconfig
 
 %changelog
+* Thu Jun 06 2024 David Morel <david.morel@vates.tech> - 1:1.1.1k-12.1
+- Sync with Centos 8 Stream's 1.1.1k-12.
+- do not package openssl.cnf file, to avoid conflict with base system openssl
+- disable weak ciphers at compile time
+- *** Upstream changelog ***
+￼- * Thu Nov 30 2023 Dmitry Belyavskiy <dbelyavs@redhat.com> - 1:1.1.1k-12
+￼- - Backport implicit rejection mechanism for RSA PKCS#1 v1.5 to RHEL-8 series
+￼-   (a proper fix for CVE-2020-25659)
+￼-   Resolves: RHEL-17696
+￼- * Wed Nov 15 2023 Clemens Lang <cllang@redhat.com> - 1:1.1.1k-11
+￼- - Fix CVE-2023-5678: Generating excessively long X9.42 DH keys or checking
+￼-   excessively long X9.42 DH keys or parameters may be very slow
+￼-   Resolves: RHEL-16538
+￼- * Thu Oct 19 2023 Clemens Lang <cllang@redhat.com> - 1:1.1.1k-10
+￼- - Fix CVE-2023-3446: Excessive time spent checking DH keys and parameters
+￼-   Resolves: RHEL-14245
+￼- - Fix CVE-2023-3817: Excessive time spent checking DH q parameter value
+￼-   Resolves: RHEL-14239
```

Comment:
There's an invisible char (seen by github, though) at the start of each of these lines.

Reply:
Indeed, that's in the merge commit, I'm trying to see if I can find a way to rebase including the merge commit to edit it, but for now, without luck.

Reply:
Don't take into account the last push force where I actually fixed it in last commit, I'll replace all that.

Reply:
Should be good: remove those, and reused the same changelog entry for the .cnf file as in 8.2.